src/main/java/gregtech/common/blocks/BlockFrame.java

```diff
@@ -303,15 +262,32 @@ public BlockFaceShape getBlockFaceShape(@Nonnull IBlockAccess worldIn, @Nonnull
         return BlockFaceShape.UNDEFINED;
     }
 
+    @Override
+    public void addInformation(ItemStack stack, @Nullable World world, List<String> tooltip, ITooltipFlag flag) {
+        if (ConfigHolder.misc.debug) {
+            tooltip.add("MetaItem Id: frame" + getGtMaterial(stack).toCamelCaseString());
```

Comment:
Should these be localized instead of hardcoded?

Reply:
It's only a debug info, I don't think there's need for localization.